### PR TITLE
New package: texstudio

### PR DESCRIPTION
**Pull request by Piraty**
- [x] report phonon compilation bug to upstream --> patched it on base of recent upstream commits
- [x] correct arm compilation (debughelper, backtrace etc.)
- [x] fix crossbuild issues by adding *-devel to hostmakedepends
- [x] fix musl + backtrace / execinfo.h issue
- [x] aarch64 compatibilty? done, backtrace function excluded

### CODE REVIEW

File: srcpkgs/texstudio/template [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+#taken from BUILD.sh
+configure_args="-unix"
+configure_args+=" -spec linux-g++"
+#configure_args+=" NO_POPPLER_PREVIEW=true"
+configure_args+=" NO_TESTS=true"
+configure_args+=" CONFIG-=debug CONFIG-=debug_and_release CONFIG+=release"
+
+#building with phonon fails so far
+#makedepends+=" phonon-qt5-devel"
+#configure_args+=" PHONON=true"
```
Comment (Vaelatern) [inline srcpkgs/texstudio/template]: Please make this a  build_option.
Why does building fail with phonon?

Comment (Vaelatern) [inline srcpkgs/texstudio/template]: I edited your comment to use ```

Comment (Vaelatern) [inline srcpkgs/texstudio/template]: Is this a known upstream problem?

Comment (Piraty) [inline srcpkgs/texstudio/template]: Thank you Vaelaern.
i will contact the project about it, but meanwhile we could use the template without phonon-support. two big distros ship texstudio without phonon support.

**Comment by Piraty** [comment]
Travis builds failed, any chance to re-run them?

**Comment by ebfe** [comment]
Could you rebase this onto current master (and squash the merge commits while at it)? That should fix the travis builds/results.

**Comment by Vaelatern** [comment]
`/void-packages/common/build-style/qmake.sh: line 5: qmake: command not found`
Isn't there a host package you can use to have qmake available in cross builds?
The musl problems are different with a lack of `backtrace`.

**Comment by Piraty** [comment]
I patched debughelper.cpp.
armv7l builds fine on my machine, armv7hf builds fine with travis.
execinfo.h needs more attention...

**Comment by Piraty** [comment]
everything builds fine now.

**Comment by Piraty** [comment]
@Vaelatern (just ping)

**Pull request merged**